README.md

```diff
@@ -10,70 +13,97 @@ pip install -e .
 ```
 
 Optionally, to run scripts, and for [wandb](https://wandb.ai) logging
+
 ```bash
 pip install -r requirements.txt
 wandb login
 ```
 
 
 ## About this repo
-This repo serves the purpose of fast prototyping [GFlowNet](https://arxiv.org/abs/2111.09266) related algorithms. It decouples the environment definition, the sampling process, and the parametrization used for the GFN loss. 
+
+This repo serves the purpose of fast prototyping [GFlowNet](https://arxiv.org/abs/2111.09266) related algorithms. It decouples the environment definition, the sampling process, and the parametrization used for the GFN loss.
 
 An example script is provided [here](scripts/train.py). To run the code, use one of the following:
+
 ```bash
-python train.py --env HyperGrid --env.ndim 4 --env.height 8 --n_iterations 100000 --loss TB 
+python train.py --env HyperGrid --env.ndim 4 --env.height 8 --n_iterations 100000 --loss TB
 python train.py --env DiscreteEBM --env.ndim 4 --env.alpha 0.5 --n_iterations 10000 --batch_size 64 --temperature 2.
-python train.py --env HyperGrid --env.ndim 2 --env.height 64 --n_iterations 100000 --loss DB --replay_buffer_size 1000 --logit_PB.module_name Uniform --optim sgd --optim.lr 5e-3 
+python train.py --env HyperGrid --env.ndim 2 --env.height 64 --n_iterations 100000 --loss DB --replay_buffer_size 1000 --logit_PB.module_name Uniform --optim sgd --optim.lr 5e-3
 python train.py --env HyperGrid --env.ndim 4 --env.height 8 --env.R0 0.01 --loss FM --optim adam --optim.lr 1e-4
 ```
 
 ### Example, in a few lines
```

Comment:
can you add a line saying you need to pip install tqdm for this ? It's not in `setup.py` (but it is in `requirements.txt`)

Reply:
According to this discussion: https://stackoverflow.com/a/33685899/3867406

Do you want tqdm to be core to your package (add to `setup.py`) or more of a deployment req (`requirements.txt`)?

Reply:
It shouldn't be core to the package. tqdm, wandb, etc... are in `requirements.txt` exactly because of this reason. They're helpful to run the accompanying scripts, but are not necessary to define / train a gflownet.